dev/packages/beats/apache-0.0.1/dataset/access/fields/package-fields.yml

```diff
@@ -1,130 +1,14 @@
+  - name: access
+    type: group
+    fields:
+    - name: user_agent
       type: group
-      fields:
-        - name: remote_ip
-          type: alias
-          path: source.address
-          migration: true
-        - name: ssl.protocol
-          type: alias
-          path: apache.access.ssl.protocol
-          migration: true
-        - name: ssl.cipher
-          type: alias
-          path: apache.access.ssl.cipher
-          migration: true
-        - name: body_sent.bytes
-          type: alias
-          path: http.response.body.bytes
-          migration: true
-        - name: user_name
-          type: alias
-          path: user.name
-          migration: true
-        - name: method
-          type: alias
-          path: http.request.method
-          migration: true
-        - name: url
-          type: alias
-          path: url.original
-          migration: true
-        - name: http_version
-          type: alias
-          path: http.version
-          migration: true
-        - name: response_code
-          type: alias
-          path: http.response.status_code
-          migration: true
-        - name: referrer
-          type: alias
-          path: http.request.referrer
-          migration: true
-        - name: agent
-          type: alias
-          path: user_agent.original
-          migration: true
-        - name: user_agent
-          type: group
-          fields:
-            - name: device
-              type: alias
-              path: user_agent.device.name
-              migration: true
-            - name: name
-              type: alias
-              path: user_agent.name
-              migration: true
-            - name: os
-              type: alias
-              path: user_agent.os.full_name
-              migration: true
-            - name: os_name
-              type: alias
-              path: user_agent.os.name
-              migration: true
-            - name: original
-              type: alias
-              path: user_agent.original
-              migration: true
-        - name: geoip
-          type: group
-          fields:
-            - name: continent_name
-              type: alias
-              path: source.geo.continent_name
-              migration: true
-            - name: country_iso_code
-              type: alias
-              path: source.geo.country_iso_code
-              migration: true
-            - name: location
-              type: alias
-              path: source.geo.location
-              migration: true
-            - name: region_name
-              type: alias
-              path: source.geo.region_name
-              migration: true
-            - name: city_name
-              type: alias
-              path: source.geo.city_name
-              migration: true
-            - name: region_iso_code
-              type: alias
-              path: source.geo.region_iso_code
-              migration: true
-    - name: error
+    - name: geoip
```

Comment:
This is interesting. We have a group with no entries. I wonder what EPM does with it 🤔 @skh @neptunian